### ISSUE TITLE
Will cachet show uptime?

### ISSUE THREAD

**Issue by vsenapathy** [fiveai/Cachet#49]
So we are reviewing applications which can show uptime of a component. Is this being thought about. Can someone let me know we are planning to include uptime for components and show historical uptime metrics

**Comment by sedan07** [comment]
@vsenapathy Yes Cachet can show metrics. These metrics could should any numerical value but certainly could show uptime say as a percentage. You just need to push in the data using the API.

**Comment by vsenapathy** [comment]
@sedan07 : But i have seen other status pages which can have uptime as part of the component. Is this in the popeline

**Comment by sedan07** [comment]
No not currently

**Comment by vsenapathy** [comment]
@sedan07 : It would def be a good feature to have. Like the component showing the uptime metrics
